<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagement</class>
 <widget class="QWidget" name="UserManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1130</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1130</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="userwidget" native="true">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#userwidget
{
background-color: rgb(83, 145, 174);
border-radius:20px;
}
</string>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1130</width>
        <height>0</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame
{
background-color: rgb(88, 88, 88);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>2</y>
         <width>111</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>用户管理</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>3</y>
         <width>28</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:14px;
image: url(:/res/funuser.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>1100</x>
         <y>3</y>
         <width>28</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华文琥珀</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 135, 49);
border-radius:14px;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnadd">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>655</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华光黑体_CNKI</family>
        <pointsize>50</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: rgb(255, 135, 49);
color: #FFFFFF;  
border-right:3px solid #e7b51e; 
border-bottom:3px solid #e7b51e;
border-left:3px solid #ffc515;
border-top:3px solid #ffc515; 
border-radius:10px;
}
QPushButton:hover
{
background-color: rgb(255, 135, 49);
color: #FFFFFF;  
border-right:3px solid #ffc515; 
border-bottom:3px solid #ffc515;
border-left:3px solid #e7b51e;
border-top:3px solid #e7b51e; 
border-radius:10px;
}
QPushButton:pressed
{
background-color: rgb(255, 135, 49);
color: #FFFFFF;  
border-right:3px solid #ffc515; 
border-bottom:3px solid #ffc515;
border-left:5px solid #e7b51e;
border-top:5px solid #e7b51e; 
border-radius:10px;
}
</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>1075</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_2
{

   border-style:outset;
   border-width:1px;
   border-color: rgb(255,255, 255);
   font:bold 30px;
   min-width:100px;
   min-height:20px;
   padding:4px;
}</string>
      </property>
      <widget class="QFrame" name="passframe">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1065</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
color:black;
   
   background-color: rgba(255, 255, 255);
   border-style:outset;
   border-width:1px;
   border-color: rgb(255,255, 255);
   font:bold 25px;
   min-width:100px;
   min-height:20px;
   padding:4px;
border-radius:10px;
}
QGroupBox
{
background-color: rgb(255, 255, 255);
   border-style:solid;
   border-width:3px;
   border-color: rgb(96, 192, 255);
   border-radius:10px;
   padding:10px;

}
QLabel
{
   border-style:solid;
   border-width:3px;
   border-color: rgb(255, 255, 255);
   border-radius:10px;
   padding:10px;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="geometry">
        <rect>
         <x>1058</x>
         <y>10</y>
         <width>15</width>
         <height>230</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>360</y>
        <width>1075</width>
        <height>270</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_3
{

   border-style:outset;
   border-width:1px;
   border-color: rgb(255,255, 255);
   font:bold 30px;
   min-width:100px;
   min-height:20px;
   padding:4px;

}</string>
      </property>
      <widget class="QFrame" name="passframe_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1065</width>
         <height>1871</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
color:black;
   
   background-color: rgba(255, 255, 255);
   border-style:outset;
   border-width:1px;
   border-color: rgb(255,255, 255);
   font:bold 25px;
   min-width:100px;
   min-height:20px;
   padding:4px;
border-radius:10px;
}
QGroupBox
{
background-color: rgb(255, 255, 255);
   border-style:solid;
   border-width:3px;
   border-color: rgb(96, 192, 255);
   border-radius:10px;
   padding:10px;

}
QLabel
{
   border-style:solid;
   border-width:3px;
   border-color: rgb(255, 255, 255);
   border-radius:10px;
   padding:10px;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar_2">
       <property name="geometry">
        <rect>
         <x>1058</x>
         <y>10</y>
         <width>15</width>
         <height>260</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>80</y>
        <width>40</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
   border-style:solid;
   border-width:1px;
   border-color: rgb(255, 255, 255);
   border-radius:10px;
	color: rgb(255, 135, 49);
border:none;
}</string>
      </property>
      <property name="text">
       <string>管
理
员
账
户</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>360</y>
        <width>40</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
   border-style:solid;
   border-width:1px;
   border-color: rgb(255, 255, 255);
   border-radius:10px;
	color: rgb(255, 135, 49);
border:none;
}</string>
      </property>
      <property name="text">
       <string>普
通
账
户</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>40</y>
        <width>549</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
   border-style:solid;
   border-width:1px;
   border-color: rgb(255, 255, 255);
   border-radius:10px;
	color: rgb(255, 135, 49);
border:none;
}</string>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>599</x>
        <y>40</y>
        <width>516</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
   border-style:solid;
   border-width:1px;
   border-color: rgb(255, 255, 255);
   border-radius:10px;
	color: rgb(255, 135, 49);
border:none;
}</string>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>1200</x>
        <y>330</y>
        <width>1050</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget
{
border-radius:10px;
background-color: rgb(88, 88, 88);
}
QToolButton
{
background-color: rgba(255, 255, 255,0);
color: rgb(255, 255, 255);
}
QToolButton:hover
{
background-color: rgb(113, 113, 113);
color: rgb(255, 255, 255);
}</string>
      </property>
      <widget class="QToolButton" name="toolDel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-top-left-radius:15px;
border-bottom-left-radius:15px;
</string>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset resource="../../res.qrc">
         <normaloff>:/res/usermange/del1.png</normaloff>:/res/usermange/del1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="toolMod">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:0px;</string>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
       <property name="icon">
        <iconset resource="../../res.qrc">
         <normaloff>:/res/usermange/mod1.png</normaloff>:/res/usermange/mod1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="toolRev">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:0px;</string>
       </property>
       <property name="text">
        <string>恢复</string>
       </property>
       <property name="icon">
        <iconset resource="../../res.qrc">
         <normaloff>:/res/usermange/rest1.png</normaloff>:/res/usermange/rest1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>635</y>
        <width>0</width>
        <height>97</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1065</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
color:black;
   
   background-color: rgba(255, 255, 255);
   border-style:outset;
   border-width:1px;
   border-color: rgb(255,255, 255);
   font:bold 20px;
   min-width:100px;
   min-height:20px;
   padding:4px;
border-radius:5px;
}
QGroupBox
{
background-color: rgb(255, 255, 255);
   border-style:solid;
   border-width:3px;
   border-color: rgb(96, 192, 255);
   border-radius:5px;

}
QLabel
{
   border-style:solid;
   border-width:3px;
   border-color: rgb(255, 255, 255);
   border-radius:5px;
}
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="lineE">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>420</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>请输入账户名</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOK">
        <property name="geometry">
         <rect>
          <x>983</x>
          <y>3</y>
          <width>80</width>
          <height>44</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>华光黑体_CNKI</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(255, 135, 49);
color: #FFFFFF;  
border-right:2px solid #ffc515; 
border-bottom:3px solid #e7b51e;
border-left:3px solid #ffc515;
border-top:3px solid #ffc515; 
border-top-left-radius:10px;
border-bottom-left-radius:10px;
}
QPushButton:hover
{
background-color: rgb(255, 135, 49);
color: #FFFFFF;  
border-right:2px solid #ffc515; 
border-bottom:3px solid #ffc515;
border-left:3px solid #e7b51e;
border-top:3px solid #e7b51e; 
border-top-left-radius:10px;
border-bottom-left-radius:10px;
}
QPushButton:pressed
{
    padding-left:3px;
    padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEE">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>5</y>
          <width>461</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>5</y>
          <width>10</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 173, 173);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>47</y>
         <width>1065</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
color:black;
   
   background-color: rgba(255, 255, 255);
   border-style:outset;
   border-width:1px;
   border-color: rgb(255,255, 255);
   font:bold 20px;
   min-width:100px;
   min-height:20px;
   padding:4px;
border-radius:5px;

}
QGroupBox
{
	background-color: rgb(255, 255, 255);
   border-style:solid;
   border-width:3px;
   border-color: rgb(96, 192, 255);
   border-radius:5px;

}
QLabel
{
   border-style:solid;
   border-width:3px;
   border-color: rgb(255, 255, 255);

}
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>管理员账户</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>10</y>
          <width>111</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>普通账户</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>171</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 170, 170);</string>
        </property>
        <property name="text">
         <string>请选择账户权限：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btncanel">
        <property name="geometry">
         <rect>
          <x>983</x>
          <y>3</y>
          <width>80</width>
          <height>44</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>华光黑体_CNKI</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(255, 135, 49);
color: #FFFFFF;  
border-right:2px solid #ffc515; 
border-bottom:3px solid #e7b51e;
border-left:3px solid #ffc515;
border-top:3px solid #ffc515; 
border-top-left-radius:10px;
border-bottom-left-radius:10px;
}
QPushButton:hover
{
background-color: rgb(255, 135, 49);
color: #FFFFFF;  
border-right:2px solid #ffc515; 
border-bottom:3px solid #ffc515;
border-left:3px solid #e7b51e;
border-top:3px solid #e7b51e; 
border-top-left-radius:10px;
border-bottom-left-radius:10px;
}
QPushButton:pressed
{
    padding-left:3px;
    padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </widget>
     </widget>
     <zorder>btnadd</zorder>
     <zorder>frame</zorder>
     <zorder>widget_2</zorder>
     <zorder>widget_3</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>widget</zorder>
     <zorder>widget_4</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserManagement</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>786</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
